Library/TimeInterval+ISO8601DateTests.swift

```diff
@@ -2,10 +2,17 @@
 import XCTest
 
 class TimeInterval_ISO8601DateTests: XCTestCase {
-  func testConvertToISO8601DateTimeString_FromTimeInterval() {
+  func testToISO8601DateTimeString() {
     let timeInterval = 1_506_897_315.0
 
     let iSO8601DateTimeString = timeInterval.toISO8601DateTimeString()
     XCTAssertEqual(iSO8601DateTimeString, "2017-10-01T22:35:15Z")
   }
+  
+  func testToISO8601DateTimeString_Future() {
+    let timeInterval = 1_680_048_000.0
+
+    let iSO8601DateTimeString = timeInterval.toISO8601DateTimeString()
+    XCTAssertEqual(iSO8601DateTimeString, "2023-03-29T00:00:00Z")
```

Comment:
just had a second thought about this. no need to declare the constant, just drop timeInterval.toISO8601...() directly into line 9 and 16.